README.md

```diff
@@ -148,3 +154,51 @@ device.
 
 NOTE: This selection is not persistent. The boot loader will fall back to its
 default boot order after reset.
+
+## Building with SWUpdate support
+
+The option `kas/opt/swupdate-example.yml` will create an image  with a
+[double copy root file system](https://sbabic.github.io/swupdate/overview.html#double-copy-with-fall-back)
+You can find the image under
+`build/tmp/deploy/images/iot2050/iot2050-image-swu-example-iot2050-debian-iot2050.wic.img`
+
+It also will create an binary for updating the system at
```

Comment:
`a` binary; sentence not finished, intermixed with the following lines supposedly?